src/accordion/accordion-toggle.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * @ngdoc directive
+ * @name bdAccordionToggle
+ * @module buildium.angular-elements.accordion
+ * @restrict A
+ *
+ * @description
+ * 
+ * Element responsible for toggling a {@link buildium.angular-elements.accordion.directive:bdAccordionGroup bdAccordionGroup} open or closed
+ *
+ * @param {Boolean} [isOpen] - is toggle open by default
+ * @param {Boolean} [disabled] - is toggle disabled
+ * @param {Function} [onChange] - change event to fire when open state changes
+ * @param {String} [openClass] - class to add when toggle is open
+ * @param {String} [disabledClass] - class to add when toggle is disabled
+ * @param {String} [toggleClassName] - changes toggle trigger to element matching this class name
+ * 
+ * @example
+ *
+    <example name="bd-accordion-toggle" module="buildium.angular-elements.accordion">
```

Comment:
turns out `accordion-toggle` doesn't quite work as expected, which can be seen by exploring this example in the docs

I've left the behavior unchanged while porting this over, but I'll open an issue for it after this merge